pkg/cmd/workspace/create/get_target.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2024 Daytona Platforms Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package create
+
+import (
+	"context"
+	"fmt"
+
+	apiclient_util "github.com/daytonaio/daytona/internal/util/apiclient"
+	"github.com/daytonaio/daytona/pkg/apiclient"
+	"github.com/daytonaio/daytona/pkg/cmd/target"
+	"github.com/daytonaio/daytona/pkg/common"
+	"github.com/daytonaio/daytona/pkg/views/target/selection"
+)
+
+type GetTargetConfigParams struct {
+	Ctx               context.Context
```

Comment:
I believe the Ctx should be the first argument of the appropriate function and then the params in an object to follow conventions.